src/config.php.dist

```diff
@@ -106,6 +106,12 @@ $config = array(
     // Input directory for locale strings
     //'engine.text.i18n.source' => './resource/language',
 
+    // Hash algo used for password recovery token
```

Comment:
I'd like to limit options for features that end users will actually want to change. The recovery key is an internal mechanism that is just expected to produce a deterministic string for a given user, so I don't think there is much value in adding configuration options for it. Let's rather use something that works on modern PHP versions out of the box.